### ISSUE TITLE
Repositioning single-tabbed studio window erroneously repositions tab buttons

### ISSUE THREAD

**Issue by maxmmyron** [rendajs/Renda#537]
https://user-images.githubusercontent.com/41480159/231712428-73fd53fd-bef5-4ae1-ab18-e6adf8e5bf28.mp4

Repositioning a studio window tab to the same spot (via repositioning a window to a half-tab size), the spacing between the existing tab button and the rest of the UI will shift a bit.

**Comment by jespertheend** [comment]
Finally found some exact steps to reproduce this.
This is in a new incognito window with a fresh project:
https://user-images.githubusercontent.com/2737650/235375732-431e7e92-8f91-4f0d-aacb-80d5dd9dfae9.mp4

**Comment by maxmmyron** [comment]
Weird. I followed your steps, and the result was ever-so-slightly different. After reload, the entity editor was joined with the build tab in the bottom section.

Still had the same issue with that upper section being unusable.


https://user-images.githubusercontent.com/41480159/235667481-ebd45976-13ab-49cc-8588-7a35df36c371.mp4

**Comment by jespertheend** [comment]
Yeah it seems like there are two separate issues both causing #600
I'm currently changing up some stuff that I think will fix this issue.

**Comment by jespertheend** [comment]
I was hoping this would get fixed with #635 but that doesn't seem to be the case.
But since this is only a visual issue now, I'll lower its priority.
